packages/utils/src/asAbortable.ts

```diff
@@ -0,0 +1,44 @@
+export class AbortError extends Error {
+    readonly code = 'AbortError'
```

Comment:
It would maybe make sense to have this code as value `ABORT`
- as it is an error code, we already know it is an error
- use uppercase notation similarly e.g. in `StreamrClientError`

Or was it so that this follows some other convention, which we prefer?

Reply:
Will do in a separate PR to follow.